task/buildah/0.2/buildah.yaml

```diff
@@ -376,6 +378,8 @@ spec:
         echo "Adding the entitlement to the build"
       fi
 
+      VOLUME_MOUNTS+=("$ADDITIONAL_VOLUME_MOUNTS")
```

Comment:
This will only work for one volume mount, and only if it looks like this `--volume=src:dest` (a space like this would break it `--volume src:dest`)

I would suggest something like this

```bash
VOLUME_MOUNTS=()

while read -r volume_mount; do
    VOLUME_MOUNTS+=("--volume=$volume_mount")
done <<< "$ADDITIONAL_VOLUME_MOUNTS"
```

Which would allow setting the volume mounts as a space-separated list

```yaml
name: ADDITIONAL_VOLUME_MOUNTS
value: |
  /foo:/foo
  /bar:/baz
```

Reply:
@chmeliik Thanks!  I have included this change in my experiments: https://github.com/konflux-ci/build-definitions/pull/1500/commits/24d61190e99ac9b1f0699cd057a2d55415492b68